app/code/Magento/InventoryCatalog/Plugin/Model/UpdateLegacyCatalogInventoryPlugin.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryCatalog\Plugin\Model;
+
+use Magento\Framework\App\ResourceConnection;
+use Magento\InventoryApi\Api\Data\ReservationInterface;
+use Magento\InventoryApi\Api\ReservationsAppendInterface;
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\CatalogInventory\Api\StockRegistryInterface;
+
+/**
+ * Plugin help to fill the legacy catalog inventory tables cataloginventory_stock_status and
+ * cataloginventory_stock_item to don't break the backward compatible.
```

Comment:
`to don't break the backward compatible.` ->
`to prevent breaking the backward compatibility`